appinventor/blocklyeditor/src/msg/en/_messages.js

```diff
@@ -901,6 +901,14 @@ Blockly.Msg.en.switch_language_to_english = {
     Blockly.Msg.LANG_LISTS_LOOKUP_IN_PAIRS_INPUT = 'look up in pairs  key %1 pairs %2 notFound %3';
     Blockly.Msg.LANG_LISTS_LOOKUP_IN_PAIRS_TOOLTIP = 'Returns the value associated with the key in the list of pairs';
 
+    // Join With Separator block
+    Blockly.Msg.LANG_LISTS_JOIN_WITH_SEPARATOR_HELPURL = '/reference/blocks/lists.html#joinwithseparator';
```

Comment:
You should add the block definition and corresponding HTML anchor to docs/reference/blocks/lists.html

Reply:
I have added the reference and provided a description along with a screenshot of the block.